### PR TITLE
Tune psf faster

### PR DESCRIPTION
**Pull request by orelgueta**
Change the algorithm to find the number of photons within a radius to be faster. For some reason also the changes to allow plotting the difference were included in this commit as well. Not sure why/how, but I guess this PR is not that large anyway so I keep as is.

### CODE REVIEW

File: applications/tune_psf.py [hunk 1]
```diff
@@ -169,6 +169,14 @@ def loadData(datafile):
         modelVersion=args.model_version,
         label=label,
     )
+    # If we want to start from different values then the one in the model:
```
Comment (GernotMaier) [inline applications/tune_psf.py:172]: Seems to be an incomplete sentence, what exactly do you mean here?

**Review by GernotMaier — APPROVED** [review]
I don't get that comment; after that good to merge.

File: applications/tune_psf.py (hunk 1 above)
Comment (orelgueta) [reply applications/tune_psf.py:172]: I guess a better sentence would be `If we want to start from values different than the ones currently in the model`. Would that be clear (I don't explain further on purpose)?

Either way the test failed for so I will fix it first.

Comment (GernotMaier) [reply applications/tune_psf.py:172]: Good.

**Comment by orelgueta** [comment]
I am merging even though the test has failed because I am sure that once merged the tests will pass. I made a mess with the various commits in different branches and different pull requests and I don't want to waste time trying to figure it out. If it fails after merging I will open a new PR.

**Pull request merged**